src/groups/GroupCard.tsx

```diff
@@ -0,0 +1,61 @@
+import _ from 'lodash/fp';
+import React, { ReactNode } from 'react';
+import { ClipboardButton } from 'src/components/ClipboardButton';
+import { GroupMenu } from 'src/groups/GroupMenu';
+import { CurrentUserGroupMembership } from 'src/libs/ajax/Groups';
+import colors from 'src/libs/colors';
+import { getLink } from 'src/libs/nav';
+import * as Style from 'src/libs/style';
+
+export const columnWidths = '1fr 30% 6rem 20px';
+
+interface GroupCardProps {
+  group: CurrentUserGroupMembership;
+  onDelete: () => void;
+  onLeave: () => void;
+}
+
+export const GroupCard = (props: GroupCardProps): ReactNode => {
```

Comment:
This component was memoized before, but that seems unnecessary unless rendering hundreds of these at once, and frequently re-sorting or filtering. Even if that is the case for someone, memoization is only a _suggestion_ for react to optimize it, not a guarantee.